packages/components/input/src/input.vue

```diff
@@ -297,6 +305,19 @@ export default defineComponent({
       calcIconOffset('suffix')
     }
 
+    const setHtmlMaxLength = (text: string) => {
+      const codeUnitsLength = text.length
+      const textLength = toArray(text).length
+      const maxlength = Number(attrs.value.maxlength)
+      const diffLength = codeUnitsLength - textLength
```

Comment:
Actually we should name this `lengthDiff` since using `diffLength` sounds a method in English

Reply:
My bad 😄. Fixed